cla_public/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -4369,6 +4381,159 @@ msgstr "Chwiliwch am gyfryngwr teulu yn eich ardal chi i ganfod a ydych yn gymwy
+msgstr "Mae gan AbilityNet gyngor ar wneud eich dyfais yn haws ei defnyddio os oes gennych chi anabledd."
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "What to do if you can’t access part of this service"
+msgstr "Beth i’w wneud os na allwch gael mynediad at ran o’r gwasanaeth hwn"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "If you need information on this service in a different format, you can contact us at %(email)s."
+msgstr "Os oes angen gwybodaeth arnoch am y gwasanaeth hwn mewn fformat gwahanol, gallwch gysylltu â ni yn %(email)s."
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "Reporting accessibility problems"
+msgstr "Rhoi gwybod am broblemau hygyrchedd"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "We’re always looking to improve the accessibility of this service. If you find any problems that aren’t listed on this page or think we’re not meeting the requirements of the accessibility regulations, contact %(email)s giving details of the issue and any assistive technology you are using."
+msgstr "Rydyn ni bob amser yn ceisio gwella hygyrchedd y gwasanaeth hwn. Os byddwch chi’n dod o hyd i unrhyw broblemau nad ydynt wedi’u rhestru ar y dudalen hon neu os ydych chi’n meddwl nad ydyn ni’n bodloni gofynion y rheoliadau hygyrchedd, cysylltwch â %(email)s gan roi manylion y broblem ac unrhyw dechnoleg gynorthwyol rydych chi’n ei defnyddio."
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "Enforcement procedure"
+msgstr "Gweithdrefn gorfodi"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "The Equality and Human Rights Commission (EHRC) is responsible for enforcing the accessibility regulations. If you’re not happy with how we respond to your complaint, contact the <a class=\"govuk-link\" href=\"https://www.equalityadvisoryservice.com/\" rel=\"external\">Equality Advisory and Support Service (<abbr>EASS</abbr>)</a>."
+msgstr "Y Comisiwn Cydraddoldeb a Hawliau Dynol sy’n gyfrifol am orfodi’r rheoliadau hygyrchedd. Os nad ydych chi’n fodlon â sut rydyn ni’n ymateb i’ch cwyn, cysylltwch â’r <a class=\"govuk-link\" href=\"http://eass-ws.custhelp.com/app/home\" rel=\"external\">Gwasanaeth Cynghori a Chefnogi Cydraddoldeb (EASS)</a>."
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "Technical information about this service’s accessibility"
+msgstr "Gwybodaeth dechnegol am hygyrchedd y gwasanaeth hwn"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "The Legal Aid Agency is committed to making its service accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (&#8470; 2) Accessibility Regulations 2018."
+msgstr "Mae’r Asiantaeth Cymorth Cyfreithiol wedi ymrwymo i wneud ei gwasanaeth yn hygyrch, yn unol â Rheoliadau Hygyrchedd Cyrff Sector Cyhoeddus (Gwefannau a Rhaglenni Symudol) (Rhif 2) 2018."
```

Comment:
Can you not use `&#8470;`for the Welsh or is `Rhif the equivalent/best you can do

Reply:
No, &#8470; is English only.  The Welsh has to be Rhif, and doesn't have a symbol.